tests/test_hips_outputs.py

```diff
@@ -0,0 +1,102 @@
+# This file is part of ci_lsstcam.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+import os
+import re
+
+from lsst.daf.butler import Butler
+from lsst.utils import getPackageDir
+from lsst.resources import ResourcePath
+
+butler = Butler(f'{os.environ["CI_LSSTCAM_DIR"]}/DATA', collections=['LSSTCam/runs/ci_lsstcam'])
+skymap = list(butler.registry.queryDatasets(datasetType='object'))[0].dataId['skymap']
+
+
+class TestHipsOutputs(unittest.TestCase):
+    """Check that HIPS outputs are as expected."""
+    def setUp(self):
+        self.butler = Butler(os.path.join(getPackageDir("ci_lsstcam"), "DATA"),
+                             instrument="LSSTCam", skymap=skymap,
```

Comment:
`skymap="lsst_cells_v1"` or it might just work without specifying the skymap at all.